RadarSDK/RadarLocationManager.m

```diff
@@ -522,7 +522,7 @@ - (void)handleLocation:(CLLocation *)location source:(RadarLocationSource)source
     [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug
                                        message:[NSString stringWithFormat:@"Handling location | source = %@; location = %@", [Radar stringForLocationSource:source], location]];
 
-    if (!location || ![RadarUtils validLocation:location]) {
+    if (!location.isValid) {
```

Comment:
will these calls work if the location is `NULL`? I think the first condition explicitly checks against empty / falsy

Reply:
Yes. It's a long-standing, but outdated, convention to use something like `if (!location || ![RadarUtils validLocation:location])`, but it's functionally equivalent to `if (!location.isValid)`, because messages sent to `nil` (i.e. functions called on a `nil` object) will always be false. See https://nshipster.com/nil/.